### PR TITLE
[IMP] Point of Sale: Improvements on docs about discounts

### PR DESCRIPTION
**Pull request by Lau-Odoo**

### CODE REVIEW

**Comment by AntoineVDV** [comment]
tag ?

File: point_of_sale/advanced_pricing_features/discounts.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+On the PoS interface, a new button appears. Use it to select a
+pricelist.
+
+.. image:: media/discounts_07.png
+   :align: center
+   :alt: View of the button to use for time-limited discounts via the pos interface
+
+If you click on it, the price is instantly updated to reflect the
+pricelist. Then, you can finalize the order.
```
Comment (kbr-odoo) [inline point_of_sale/advanced_pricing_features/discounts.rst:103]:
```suggestion
Now click on it to instantly updated the prices with the
pricelist selected. Then, you can finalize the order.
```
Something like this maybe. I just tried to "continue" the explanation considering the paragraph before.

File: point_of_sale/advanced_pricing_features/discounts.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+Create a pricelist
+------------------
+
+By default, Odoo has a *Public Pricelist* configured. To create more,
+go to :menuselection:`Point of Sale --> Products --> Pricelists`. Then
+click on create.
+
+When creating a pricelist, you can set several criteria to use a
+specific price: period, min. quantity, etc. You can also decide to apply
```
[the remark this answers is not in the document]
Comment (Lau-Odoo) [reply point_of_sale/advanced_pricing_features/discounts.rst:84]: I let you decide which version you prefer @jubodoo

File: point_of_sale/advanced_pricing_features.rst [hunk 3]
```diff
@@ -5,8 +5,7 @@ Pricing Features
 .. toctree::
    :titlesonly:
 
-   advanced_pricing_features/manual_discount
-   advanced_pricing_features/seasonal_discount
+   advanced_pricing_features/discounts
```
Comment (AntoineVDV) [inline point_of_sale/advanced_pricing_features.rst:8]: Two redirect rules are needed here

**Comment by Lau-Odoo** [comment]
The redirect.rst file has been updated.
This doc is waiting for your review @jubodoo

**Pull request merged**